### PR TITLE
feat: webhooks management for alerts and schedules

### PR DESCRIPTION
**Pull request by hackerstanislav**
Tests for new added org SPI

JIRA: F1-349

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18975/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20757/ : SUCCESS in 9m 50s

File: libs/sdk-backend-tiger/src/convertors/fromBackend/NotificationChannelsConvertor.ts [hunk 1]
```diff
@@ -15,7 +15,9 @@ export function convertWebhookFromNotificationChannel(webhook: INotificationChan
         triggers:
             webhook.attributes?.triggers?.map((trigger) => ({
                 type: trigger.type,
-                allowOn: (trigger.metadata as any).allowedOn ?? [],
+                ...((trigger.metadata as any)?.allowedOn
```
Comment (kandl) [inline libs/sdk-backend-tiger/src/convertors/fromBackend/NotificationChannelsConvertor.ts:18]: Should this be covered by some typeguard / discriminated union?

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18976/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20758/ : SUCCESS in 13m 42s

**Pull request merged**